post.md

```diff
@@ -0,0 +1,427 @@
+# Server Actions with User Confirmations in Next.js 13
```

Comment:
For easier searches (SEO), I'd recommend a title with more active voice. Something like, "How to use Next.js 13 Server Actions".